tesseract/tesseract_environment/test/tesseract_environment_unit.cpp

```diff
@@ -10,6 +10,7 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 #include <tesseract_environment/core/types.h>
 #include <tesseract_environment/kdl/kdl_state_solver.h>
+#include <tesseract_environment/ofkt/ofkt_state_solver.h>
```

Comment:
Outside the scope of this PR, but it might be interesting to make some benchmarks for the state solvers like we have for collision